addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -55,9 +55,13 @@
             <field name="arch" type="xml">
                 <form string="Mailing">
                     <header style="min-height:31px;">
-                        <button name="action_put_in_queue" type="object" attrs="{'invisible': [('state', 'in', ('in_queue', 'done'))]}" class="oe_highlight" string="Send"
+                        <button name="action_put_in_queue" type="object" attrs="{'invisible': ['|', ('state', 'in', ('in_queue', 'done')), ('schedule_type', '=', 'scheduled')]}"
```

Comment:
We usually put the id first, then name, then the rest (TODO for both).

Reply:
Done :+1: